vendor/bat-native-ads/src/bat/ads/internal/account/redeem_unblinded_token/redeem_unblinded_token.cc

```diff
@@ -94,18 +94,19 @@ void RedeemUnblindedToken::OnCreateConfirmation(
   BLOG(6, UrlResponseToString(url_response));
   BLOG(7, UrlResponseHeadersToString(url_response));
 
-  if (url_response.status_code == net::HTTP_BAD_REQUEST) {
-    // OnFetchPaymentToken handles HTTP response status codes for duplicate/bad
-    // confirmations as we cannot guarantee if the confirmation was created or
-    // not, i.e. after an internal server error 500
-    BLOG(1, "Duplicate/bad confirmation");
-  }
+  if (confirmation.credential.empty()) {
+    if (url_response.status_code == 418) {  // I'm a teapot
```

Comment:
If the credential is empty we are only sending a confirmation and not redeeming an unblinded token